provider/objectstorage_bucket_resource_test.go

```diff
@@ -56,6 +57,24 @@ func (s *ResourceObjectstorageBucketTestSuite) TestAccResourceObjectstorageBucke
 					resource.TestCheckResourceAttr(s.ResourceName, "metadata.content-type", "text/plain"),
 				),
 			},
+			// test update
+			{
+				Config: s.Config + tokenFn(`
+				resource "oci_objectstorage_bucket" "t" {
+					namespace = "${data.oci_objectstorage_namespace.t.namespace}"
+					compartment_id = "${var.compartment_id}"
+					name = "{{.token}}-changed"
```

Comment:
Does anyone know why "token" is used in the original test case (see the line 48 above)?  I kept using it here just in case, but I don't understand the intention why the bucket name needs to be dynamically generated for every run.

Reply:
It needs to be unique per test to prevent collisions with parallel tests or buckets that were not cleaned up from previous tests (if they had failed). It is `tokenize()` above that generates a new token and corresponding `tokenFn`. `tokenFn` just makes sure the same token is injected into each template string across multiple steps.

Reply:
Thanks Cody for the clarification!